BlazarUI/app/scripts/components/sidebar/SidebarContainer.jsx

```diff
@@ -58,16 +60,28 @@ class SidebarContainer extends Component {
   componentDidMount() {
     this.unsubscribeFromBuilds = BuildsStore.listen(this.onStoreChange);
     this.unsubscribeFromStars = StarStore.listen(this.onStarChange);
-    BuildsActions.loadBuilds(this.props.params);
+    this.startPollingBuilds();
     window.addEventListener('resize', this.handleResizeDebounced);
+    window.addEventListener('blur', this.stopPollingBuilds);
+    window.addEventListener('focus', this.startPollingBuilds);
   }
-  
+
   componentWillUnmount() {
     BuildsActions.stopPollingBuilds();
     this.unsubscribeFromBuilds();
     window.removeEventListener('resize', this.handleResizeDebounced);
+    window.removeEventListener('blur', this.stopPollingBuilds);
+    window.removeEventListener('focus', this.startPollingBuilds);
+  }
+
+  startPollingBuilds() {
```

Comment:
I created functions to call these reflux actions so that `add/removeEventListener` had something named to refer to.